_spinnaker/pacrd.md

```diff
@@ -0,0 +1,372 @@
+grepping for `applications.pacrd.armory.spinnaker.io` in the installation
+manifest. Fields are documented under `spec.validation.openAPIV3Schema`.
+
+### Creating
+
+Applications in Kubernetes can be defined with the same options you would define
+in the UI. The following example shows how you might define a simple Application:
+
+```yaml
+# file: application.yaml
+apiVersion: pacrd.armory.spinnaker.io/v1alpha1
+kind: Application
+metadata:
+  name: myapplicationname
+spec:
+  email: sam.tribal@armory.io
+  description: My Application is a catalogue of Widgets surfaced by an API.
+```
+
+*Note: Application names must adhere to both [Kubernetes][kube-name-doc]
+__and__ Spinnaker name standards.*
+
+If you've defined this in an `application.yaml` file then you can create it in
+your cluster by running the following `kubectl` command:
+
+```
+kubectl apply -f application.yaml
+```
+
+Once applied, you can check on the status of your application by using either
+the `get` or `describe` commands. `kubectl` will recognize either `app` or
+`application` for the resource kind:
+
+```sh
+❯ kubectl get app myapplicationname
+NAME                URL                                                             LASTCONFIGURED   STATUS
+myapplicationname   http://spinnaker.io/#/applications/myapplicationname/clusters   7m26s            Created
+
+# or ...
+
+❯ kubectl get application myapplicationname
+NAME                URL                                                             LASTCONFIGURED   STATUS
+myapplicationname   http://spinnaker.io/#/applications/myapplicationname/clusters   7m26s            Created
+```
+
+### Updating
+
+Applications can be updated in one of two ways:
+
+1. Re-applying the application manifest in your repository
+    - `kubectl apply -f application.yaml`
+1. Editing the application manifest in-cluster
+    - `kubectl edit app myapplicationname`
+
+In both cases, when an application has been updated in your Kubernetes cluster,
+you will see changes propagate into Spinnaker shortly. If an error occurred
+during the update your application may show an `ErrorFailedUpdate` state. You
+can see the details of that failure by describing the resource and looking in
+the "Events" section:
+
+```
+kubectl describe app myapplicationname
+```
+
+### Deleting
+
+When an application is no longer necessary, you can delete it in one of two ways:
+
+1. Re-applying the application manifest in your repository
+  - `kubectl delete -f application.yaml`
+1. Deleting the application directly
+  - `kubectl delete app myapplicationname`
+
+In either case the application will be deleted shortly in Spinnaker as well. If
+an error occurred during deletion then your application may show an
+`ErrorFailedDelete` state. You can see the details of that failure by
+describing the resource and looking in the "Events section":
+
+```
+kubectl describe app myapplicationname
+```
+
+#### Caveats
```

Comment:
Is there a reason why Caveats exist in addition to Current Limitations? Can they be combined into one "Known Issues and Limitations" section at the bottom of near the start of the doc?

eg

## Known Issues and Limitations

**Applications**
* asdlkjfdsklaj;f

**Pipelines**
* aslkd;jfasidfj

Reply:
I can consolidate them